test/e2e/sample_e2e_test.go

```diff
@@ -114,12 +114,11 @@ var _ = Describe("Sample", func() {
 
 			// TODO: do we need to create catalog images are each scenario ? can we move building images to once per test suite (group of tests that uses the catalog)
 			// don't want to rebuild the image if it already exists
-			operatorImage := "quay.io/cdjohnson/ibm-sample-panamax-operator@sha256:47c9fcef261f3f26dbb6c05f904d11563aae43f61d72e0e4556399f3003c97d0"
-			operatorCommand := []string{"ibm-sample-panamax-operator"}
+			operatorImage := *dummyImage
 			catalogEntry := []cu.CatalogEntry{
-				{Version: semver.MustParse("1.0.0"), ReplacesVersion: "", SkipRange: "", DefaultChannel: "alpha", Channels: []string{"alpha"}, NewIndex: true, PackageName: "testoperatora", OwnedGVKs: cu.A1v1CRDDescription, DependencyGVKs: nil, DependencyPackages: nil, Addmode: cu.SemverSkipPatch, ConfigMap: nil, Secret: nil, CrdVersions: cu.V1CRDVersionV1beta1, OperatorImage: operatorImage, OperatorCommand: operatorCommand},
-				{Version: semver.MustParse("1.0.1"), ReplacesVersion: "", SkipRange: "<1.0.1", DefaultChannel: "alpha", Channels: []string{"alpha"}, NewIndex: false, PackageName: "testoperatora", OwnedGVKs: cu.A1v1CRDDescription, DependencyGVKs: nil, DependencyPackages: nil, Addmode: cu.SemverSkipPatch, ConfigMap: nil, Secret: nil, CrdVersions: cu.V1CRDVersionV1beta1, OperatorImage: operatorImage, OperatorCommand: operatorCommand},
-				{Version: semver.MustParse("1.0.0"), ReplacesVersion: "", SkipRange: "", DefaultChannel: "alpha", Channels: []string{"alpha"}, NewIndex: false, PackageName: "testoperatorb", OwnedGVKs: cu.B1v1CRDDescription, DependencyGVKs: cu.A1v1CRDDescription, DependencyPackages: nil, Addmode: cu.SemverSkipPatch, ConfigMap: nil, Secret: nil, CrdVersions: cu.V1CRDVersionV1beta1, OperatorImage: operatorImage, OperatorCommand: operatorCommand},
+				{Version: semver.MustParse("1.0.0"), ReplacesVersion: "", SkipRange: "", DefaultChannel: "alpha", Channels: []string{"alpha"}, NewIndex: true, PackageName: "testoperatora", OwnedGVKs: cu.A1v1CRDDescription, DependencyGVKs: nil, DependencyPackages: nil, Addmode: cu.SemverSkipPatch, ConfigMap: nil, Secret: nil, CrdVersions: cu.V1CRDVersionV1beta1, OperatorImage: operatorImage, OperatorCommand: nil},
```

Comment:
There is no operator command for the dummy nginx image, so changed to nil

Reply:
There is a default command in the image:

```
        "Cmd": [
            "/bin/sh",
            "-c",
            "#(nop) ",
            "CMD [\"/opt/bitnami/scripts/nginx/run.sh\"]"
        ],
```

However I agree that there's no reason to override the default so nil makes sense here.

Reply:
not sure this is right... the default dummy image `bitnami/nginx:latest` is not an operator... its an operand, so this will need to be in sync with whatever we use for an operator

Reply:
If we do stick with the dummy image `bitnami/nginx:latest` then it's probably ok to have this as nil since we don't need to override the default command that is included in this image:
```
        "Cmd": [
            "/bin/sh",
            "-c",
            "#(nop) ",
            "CMD [\"/opt/bitnami/scripts/nginx/run.sh\"]"
        ],
```

Reply:
> not sure this is right... the default dummy image bitnami/nginx:latest is not an operator... its an operand, so this will need to be in sync with whatever we use for an operator

I think there's slight difference here. Its true for the panamax example, the nginx is the operand. But the existing set of tests in the OLM treat the dummyImage(nginx) as the operator, [example](https://github.com/operator-framework/operator-lifecycle-manager/blob/1cc5b19ed58d61a5da717402ee03ed9053408fd8/test/e2e/subscription_e2e_test.go#L2248)